src/System.IO/src/System/IO/TextWriter.cs

```diff
@@ -597,5 +598,332 @@ public override void Write(char value)
             {
             }
         }
+
+#if netstandard17
```

Comment:
Should these be define in their own conditionally-compiled file rather than using ifdefs?